authentik/root/db/base.py

```diff
@@ -1,5 +1,5 @@
 """authentik database backend"""
-from django_prometheus.db.backends.postgresql.base import DatabaseWrapper as BaseDatabaseWrapper
+from django_tenants.postgresql_backend.base import DatabaseWrapper as BaseDatabaseWrapper
```

Comment:
I wonder if we can combine these somehow to still get the prometheus metrics

Reply:
This is still done in here: https://github.com/goauthentik/authentik/blob/multi-tenant-django-tenants/authentik/root/settings.py#L285:
```
ORIGINAL_BACKEND = "django_prometheus.db.backends.postgresql"
```
django-tenants will pick-up that backend and depend from it: https://github.com/django-tenants/django-tenants/blob/master/django_tenants/postgresql_backend/base.py#L60